src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -136,16 +136,20 @@ public string Accept
 
         public virtual bool AllowReadStreamBuffering
         {
+            // Same behavior as .NET Framework.
```

Comment:
I can remove it but I thought it was helpful to future reviewers when they look at this code and try to figure out if NotSupport/NotImplemented was intentional or just a temporary thing. I will clean those up on next PR with other test fixes.